services/src/integ-test/resources/uk/gov/moj/sdt/services/sql/executeScriptTest1.sql

```diff
@@ -1,3 +1,7 @@
+TRUNCATE TABLE BULK_CUSTOMERS CASCADE;
+TRUNCATE TABLE SERVICE_REQUESTS CASCADE;
```

Comment:
No test data is inserted into the SERVICE_REQUESTS table by this script.  Should this be truncating the TARGET_APPLICATIONS table instead?